### PR TITLE
Favorites factory BF-026

### PR DESCRIPTION
**Pull request by Migueld1324**

### CODE REVIEW

**Review by devRodSoft — APPROVED** [review]

File: blimflix/client/app/providers/favorites.factory.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  /* @ngInject */
+  function favorites($resource) {
+    /**
+     * Accessible members at top
+     *   https://github.com/johnpapa/angular-styleguide#style-y052
+     */
+    return $resource('http://localhost:3333/users/:user_id/favorites/:movie_id',
+      {user_id: '@user_id', movie_id: '@movie_id'},
+      {
+        'get'   : { method: 'GET', isArray: true, },
```
Comment (CarlosPadilla) [inline blimflix/client/app/providers/favorites.factory.js]: could be better use the same convention for all the resourses
the standard convention is
```
query : GET an array of elements
get : GET just one of these elements
save: POST create a new element
delete: DELETE one of these elements
remove: same as delete
update: PUT update one of these elements
```

**Review by CarlosPadilla — CHANGES_REQUESTED** [review]

File: blimflix/client/app/providers/favorites.factory.js (hunk 1 above)
Comment (Migueld1324) [reply blimflix/client/app/providers/favorites.factory.js]: Ok, working on that!

Comment (Migueld1324) [reply blimflix/client/app/providers/favorites.factory.js]: so, it has to be query, save and remove or delete in this case, right?

**Review by CarlosPadilla — APPROVED** [review]
That's right

**Pull request merged**